src/org/centos/pipeline/PipelineUtils.groovy

```diff
@@ -40,16 +42,16 @@ def setupStage(stage, sshKey) {
 }
 
 /**
- * Library to rsync data back to artifacts.ci.centos.org
+ * Library to execute a task and rsync the logs back to artifacts.ci.centos.org
  *
  * variables
  *  stage - current stage running
  *  duffyKey - duffy file credential name stored in Jenkins credentials
  */
-def rsyncResults(stage, duffyKey) {
-    echo "Currently in stage: ${stage} in rsyncResults"
+def runTaskAndReturnLogs(stage, duffyKey) {
+    echo "Currently in stage: ${stage} in runTaskAndReturnLogs"
 
-    withCredentials([file(credentialsId: 'duffy-key', variable: 'DUFFY_KEY'), file(credentialsId: 'fedora-keytab', variable: 'FEDORA_KEYTAB')]) {
+    withCredentials([file(credentialsId: duffyKey, variable: 'DUFFY_KEY'), file(credentialsId: 'fedora-keytab', variable: 'FEDORA_KEYTAB')]) {
```

Comment:
I thought you confirmed this with me in irc.  it's duffy-key

Reply:
I did. This usage of duffyKey is an argument to the method. Having the method parameterized could be useful in case the name of the key in the Jenkins credentials store changed?

Reply:
Lets not modify Jenkinsfile since that should be stable and we will swap out for the shared libraries.   Thanks

Reply:
@arilivigni not sure what you mean?